src/core/state/favorites/index.ts

```diff
@@ -120,6 +128,18 @@ const defaultFavorites = {
     APEUSD_APECHAIN_ADDRESS,
   ],
   [ChainId.ink]: [ETH_INK_ADDRESS, WETH_INK_ADDRESS],
+  [ChainId.sanko]: [
+    DMT_SANKO_ADDRESS,
+    WDMT_SANKO_ADDRESS,
+    USDC_SANKO_ADDRESS,
+    WETH_SANKO_ADDRESS,
+  ],
+  [ChainId.gravity]: [
```

Comment:
does `WG_GRAVITY_ADDRESS` need to be included here as well?

Reply:
Yup good catch